SharpPcap/Pcap.cs

```diff
@@ -68,6 +70,35 @@ public static string Version
             }
         }
 
+        private static Version _libpcapVersion;
+        public static Version LibpcapVersion
+        {
+            get
+            {
+                _libpcapVersion = _libpcapVersion ?? GetLibpcapVersion(Version);
+                return _libpcapVersion;
+            }
+        }
+
+        public static Version SharpPcapVersion
```

Comment:
@chmorgan having a class called `SharpPcap.Version` will most of the time conflict the the `System.Version` class.
So I moved it here.

Reply:
@kayoub5 the commit log says you are adding support for detecting the libpcap version, you should also mention in the detailed log why you are moving the SharpPcap version as well. I guess its ok being in the Pcap class but it does read a little odd to say Pcap.xxx vs. SharpPcap.xxx to access things. Is the version overlap really a thing? I couldn't find any guidance on not doing it but agree that if people are 'using System' and 'using SharpPcap' you'll end up with an overlap.

Reply:
Overlap in class names is at least annoying especially when it conflicts with a very commonly used namespace such as `System`.

I updated the commit message to mention the change.

Reply:
k. If this is better than lets move it.